lib/reporter/cli.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const chalk = require('chalk');
+
+const report = module.exports = {};
+
+// Pa11y version support
+report.supports = '^5.0.0 || ^5.0.0-alpha || ^5.0.0-beta';
+
+// Helper strings for use in reporter methods
+const start = chalk.cyan(' > ');
+const typeStarts = {
+	error: chalk.red(' • Error:'),
+	notice: chalk.cyan(' • Notice:'),
+	unknown: chalk.grey(' •'),
+	warning: chalk.yellow(' • Warning:')
+};
+
+// Output the welcome message once Pa11y begins testing
+report.begin = () => {
+	return cleanWhitespace(`
+
```

Comment:
the irony of having so much whitespace in a method called `cleanWhitespace` is really making me giggle